### ISSUE TITLE
Sonar RSPEC-1192

### ISSUE THREAD

**Issue by mirecl** [charliermarsh/ruff#3574]
Hi all!
Tell me pls - now there is a check for duplicate value?

_For example in Sonar: https://rules.sonarsource.com/python/tag/design/RSPEC-1192_

**Comment by charliermarsh** [comment]
I don't think so -- we don't have a check like this, though we could add a Sonar category with this rule (and others) if someone were interested.

**Comment by mike1936** [comment]
Voting up for adding Sonar category. Also, I'm curious if there is any plan to add a tag system in ruff in the near future, similar to what Sonar offers.

**Comment by Avasam** [comment]
Linking to general issue for Sonar rules: #4935

**Comment by hafiz-bs23** [comment]
Also voting up for adding Sonar category
